mitodl/ocw-course-hugo-theme
# update docker config to support config folder

#### Pre-Flight checklist


#### What are the relevant tickets?
Related to https://github.com/mitodl/ocw-to-hugo/issues/229

#### What's this PR do?
Moving forward, we are going to want to decentralize our Hugo configuration and allow for parts to be introduced piecemeal. 
 This PR updates the docker configuration for running a course site locally to use the default Hugo `configDir` of `config/_default`.  It creates the folder and copies the default `config.toml` in there, then merges in any config included with the course in the `ocw-to-hugo` output.

#### How should this be manually tested?
 - Clone `ocw-to-hugo` and check out the `cg/external-links` branch
 - Read the readme for this repo and configure it to run locally in Docker, point it at your local `ocw-to-hugo`
 - Spin up a local site for `7-00-covid-19-sars-cov-2-and-the-pandemic-fall-2020`
 - Verify that there is a link in the nav titled "ONLINE PUBLICATION" that links to https://biology.mit.edu/undergraduate/current-students/subject-offerings/covid-19-sars-cov-2-and-the-pandemic/

#### Any background context you want to provide?
This PR was made to support changes made by https://github.com/mitodl/ocw-to-hugo/pull/239

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/12089658/112541023-4144dc80-8d89-11eb-93ec-811384015667.png)
![image](https://user-images.githubusercontent.com/12089658/112541105-5f124180-8d89-11eb-8c83-076643b05efd.png)
